### PR TITLE
update READMEs to list libraries in a more sensible fashion since they all use web5

### PR DESCRIPTION
**Pull request by finn-tbd**
Based on a [request by from a PR in a different repo](https://github.com/TBD54566975/developer.tbd.website/pull/1449#issuecomment-2189404101)

### CODE REVIEW

File: javascript/README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-| dinger                 | web5-js | A progressive web app to show DID to DID communication |
-| fan-club-credential    | web5-js | demonstrates how to use the @web5/credentials package to issue and verify Verifiable Credentials (VCs) |
-| kcc-prototype-exemplar | web5-js | a simple implementation to illustrate the [known-customer-credential](https://github.com/TBD54566975/known-customer-credential) specification |
-| pfi-aud-usd-tbdex      | tbdex-js, web5-js |  an example remittance PFI implementation using the tbdex protocol which uses a real payment API |
-| pfi-guide-example      | tbdex-js | ?? |
-| shared-todo            | web5-js  | a simple Web5 app to manage a set of todos towards your goals with friends |
-| tbdex-pfi-exemplar     | tbdex-js, web5-js | a starter kit for building a **Participating Financial Institution (PFI)** gateway to provide liquidity services on the **[tbDEX](https://developer.tbd.website/projects/tbdex/) network**. |
-| todo                   | web5-js  | a todo app uses the Web5 JS SDK to create a DWN in your browser and add todos |
+| Name                   | TBDex? | Description |
```
Comment (angiejones) [inline javascript/README.md:5]: correct case is `tbDEX`

File: javascript/README.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+|------------------------|--------|-------------|
+| book-reviews           | no     | A book review app using Web5. |
+| dinger                 | no     | A progressive web app to show DID to DID communication |
+| fan-club-credential    | no     | demonstrates how to use the @web5/credentials package to issue and verify Verifiable Credentials (VCs) |
+| kcc-prototype-exemplar | no     | a simple implementation to illustrate the [known-customer-credential](https://github.com/TBD54566975/known-customer-credential) specification |
+| pfi-aud-usd-tbdex      | yes    |  an example remittance PFI implementation using the tbdex protocol which uses a real payment API |
```
Comment (angiejones) [inline javascript/README.md:11]: using the `tbDEX` protocol

File: kotlin/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # TBD Kotlin Examples
 
-These examples use TBD's Kotlin libraries ([tbdex-kt](https://github.com/TBD54566975/tbdex-kt) or [web5-kt](https://github.com/TBD54566975/web5-kt)).
+These examples use TBD's Kotlin libraries ([web5-kt](https://github.com/TBD54566975/web5-kt) and sometimes [tbdex-kt](https://github.com/TBD54566975/tbdex-kt)).
 
-| Name                   | Libraries Used | Description |
-|------------------------|----------|-------------------|
-| tbdex-example-android  | tbdex-kt | shows the tbdex SDK working in Android in a simple mobile app with Kotlin. |
+| Name                   | TBDex? | Description |
```
Comment (angiejones) [inline kotlin/README.md:5]: tbDEX

File: swift/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # TBD Swift Examples
 
-These examples use TBD's Swift libraries ([tbdex-swift](https://github.com/TBD54566975/tbdex-swift) or [web5-swift](https://github.com/TBD54566975/web5-swift)).
+These examples use TBD's Swift libraries ([web5-swift](https://github.com/TBD54566975/web5-swift) and sometimes [tbdex-swift](https://github.com/TBD54566975/tbdex-swift)).
```
Comment (angiejones) [inline swift/README.md:3]: can we remove the possessive `TBD's`? all instances. they are open source libraries and dont belong to TBD

File: swift/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
 
-| Name                   | Libraries Used | Description |
-|------------------------|----------------|-------------|
-| tbdex-example-ios      | tbdex-swift    | example iOS application that communicates with the [tbdex-pfi-examplar](https://github.com/TBD54566975/tbdex-pfi-exemplar). |
+| Name                   | TBDex? | Description |
```
Comment (angiejones) [inline swift/README.md:5]: tbDEX

File: swift/README.md (hunk 4 above)
Comment (finn-tbd) [reply swift/README.md:3]: What term should we use to describe these libraries that doesn't include all other random dependencies? is "web5" better?

**Pull request closed without merging**